.mergify.yml

```diff
@@ -238,37 +238,37 @@ pull_request_rules:
       - "check-success=crds-gen"
       - "check-success=docs-check"
       - "check-success=pylint"
-      - "check-success=canary (quay.io/ceph/ceph:v18)"
-      - "check-success=raw-disk-with-object (quay.io/ceph/ceph:v18)"
-      - "check-success=two-osds-in-device (quay.io/ceph/ceph:v18)"
-      - "check-success=osd-with-metadata-partition-device (quay.io/ceph/ceph:v18)"
-      - "check-success=osd-with-metadata-device (quay.io/ceph/ceph:v18)"
-      - "check-success=encryption (quay.io/ceph/ceph:v18)"
-      - "check-success=lvm (quay.io/ceph/ceph:v18)"
-      - "check-success=pvc (quay.io/ceph/ceph:v18)"
-      - "check-success=pvc-db (quay.io/ceph/ceph:v18)"
-      - "check-success=pvc-db-wal (quay.io/ceph/ceph:v18)"
-      - "check-success=encryption-pvc (quay.io/ceph/ceph:v18)"
-      - "check-success=encryption-pvc-db (quay.io/ceph/ceph:v18)"
-      - "check-success=encryption-pvc-db-wal (quay.io/ceph/ceph:v18)"
-      - "check-success=encryption-pvc-kms-vault-token-auth (quay.io/ceph/ceph:v18)"
-      - "check-success=encryption-pvc-kms-vault-k8s-auth (quay.io/ceph/ceph:v18)"
-      - "check-success=lvm-pvc (quay.io/ceph/ceph:v18)"
-      - "check-success=multi-cluster-mirroring (quay.io/ceph/ceph:v18)"
-      - "check-success=rgw-multisite-testing (quay.io/ceph/ceph:v18)"
-      - "check-success=encryption-pvc-kms-ibm-kp (quay.io/ceph/ceph:v18)"
-      - "check-success=multus-cluster-network (quay.io/ceph/ceph:v18)"
-      - "check-success=csi-hostnetwork-disabled (quay.io/ceph/ceph:v18)"
+      - "check-success=canary"
+      - "check-success=raw-disk-with-object"
+      - "check-success=two-osds-in-device"
+      - "check-success=osd-with-metadata-partition-device"
+      - "check-success=osd-with-metadata-device"
+      - "check-success=encryption"
+      - "check-success=lvm"
+      - "check-success=pvc"
+      - "check-success=pvc-db"
+      - "check-success=pvc-db-wal"
+      - "check-success=encryption-pvc"
+      - "check-success=encryption-pvc-db"
+      - "check-success=encryption-pvc-db-wal"
+      - "check-success=encryption-pvc-kms-vault-token-auth"
+      - "check-success=encryption-pvc-kms-vault-k8s-auth"
+      - "check-success=lvm-pvc"
+      - "check-success=multi-cluster-mirroring"
+      - "check-success=rgw-multisite-testing"
+      - "check-success=encryption-pvc-kms-ibm-kp"
+      - "check-success=multus-cluster-network"
+      - "check-success=csi-hostnetwork-disabled"
```

Comment:
I'm comparing the test from [4.13](https://github.com/rook/rook/actions/runs/8663216879) and [4.14](https://github.com/rook/rook/actions/runs/8754863375). Based on the differences, I have a suggestion for these to work with 4.14.

I think 4.14 tests should be `canary-tests / <test-name> (<matrix-val>)`. I think the space around the `/` is the right one given that the 4.14 tests seem to have the space, but I'm not certain about that. I can't seem to find any docs about it.

So `canary-tests / canary (quay.io/ceph/ceph:v18)`, etc.

Reply:
I copied `canary-tests / lvm-pvc (quay.io/ceph/ceph:v18)` from the bottom of this page: https://github.com/rook/rook/actions/runs/8754863375. 

I think the space around the slash is correct.

Reply:
Good point! I overlooked that the tests were all under the `canary-tests` node. I'll try that.